mousestyles/classification/classification.py

```diff
@@ -90,20 +96,28 @@ def fit_random_forest(train_y, train_x, test_x,
         es = [500, 100]
     if max_feature is None:
         fs = [10, 20, 30, 40, 50, 60, 70, 80, 90, 99]
+    if importance_level is None:
+        importance_level = 0.05
     rf = RandomForestClassifier()
     clf = GridSearchCV(
         estimator=rf, param_grid=dict(n_estimators=es, max_features=fs))
     clf.fit(train_x, train_y)
     clf = clf.best_estimator_
+    imp = clf.feature_importances_
+    important_feature = []
+    names = test_x.columns
+    for i in range(len(imp)):
+        if sorted(zip(imp, names))[i][0] >= importance_level:
+            important_feature.append(sorted(zip(imp, names))[i])
```

Comment:
wondering if it's better to assign the sorted zip to a variable before the for loop. currently, for each feature, we are performing the same sorting twice. alternatively, using a boolean to slicing a np.ndarray might also work.